todo_app/app.py

```diff
@@ -8,11 +8,23 @@
 
 from todo_app.flask_config import Config
 from todo_app.data.viewmodel import ViewModel
+import logging
 
+from loggly.handlers import HTTPSHandler
```

Comment:
This doesn't look to be installed currently - it should be added to your dependencies with `poetry add loggly-python-handler`

Reply:
done